build.gradle

```diff
@@ -0,0 +1,265 @@
+
+        publishing {
+            publications {
+                maven(MavenPublication) {
+                    artifact ear
+                }
+            }
+            repositories {
+                maven {
+                    name "releases.maven.research.rackspace.com"
+                    url "http://maven.research.rackspacecloud.com/content/repositories/releases"
+                }
+            }
+        }
+    }
+
+}
+import org.apache.tools.ant.filters.ReplaceTokens
+
+project(':impersonation-filter-tests'){
+
+    apply plugin: 'groovy'
+    apply plugin: 'org.unbroken-dome.test-sets'
+    //apply plugin: 'ear'
+
+    def spockVersion = "1.0-groovy-2.4"
+
+    repositories {
+        flatDir {
+            name 'localRepository'
+        }
+        maven { url "http://maven.research.rackspacecloud.com/content/repositories/releases" }
+        localRepository {
+            dirs 'lib'
+        }
+    }
+
+    dependencies {
+        testCompile("org.openrepose:core-lib:${this.properties['reposeVersion']}")
+        testCompile("org.apache.commons:commons-lang3:3.3.2")
+        testCompile("org.codehaus.groovy:groovy:2.4.4")
+        testCompile("junit:junit:4.12")
+        testCompile("org.apache.httpcomponents:httpclient:4.5.1")
+        testCompile("org.spockframework:spock-core:$spockVersion")
+        testCompile("org.spockframework:spock-spring:$spockVersion")
+        testCompile("org.springframework:spring-test:4.1.4.RELEASE")
+        testCompile("org.apache.logging.log4j:log4j-core:2.3")
+        testCompile("org.linkedin:org.linkedin.util-groovy:1.8.0")
+        testCompile("org.rackspace:deproxy:0.21")
+        runtime("org.openrepose:valve:${this.properties['reposeVersion']}@jar")
+        runtime "org.openrepose:filter-bundle:${this.properties['reposeVersion']}@ear"
+        runtime "org.openrepose:extensions-filter-bundle:${this.properties['reposeVersion']}@ear"
+    }
+
+    task initConfig(type: Copy) {
+        from('src/test/resources') {
+            include '**/*.properties'
+            filter(ReplaceTokens, tokens: [
+                    "repose.version": this.properties['reposeVersion'],
+                    "repose.valve.jar": "valve-" + this.properties['reposeVersion'] + ".jar",
+                    "repose.container": "valve",
+                    "build.dir": buildDir.absolutePath,
+                    "repose.home": buildDir.absolutePath + "/repose_home",
+                    "target.hostname": "localhost",
+                    "repose.config.templates": buildDir.absolutePath + "/configs"
+            ])
+        }
+        into 'build/resources/test'
+
+        outputs.upToDateWhen { false }
+    }
+
+    task copyConfigs(type: Copy) {
+        logger.warn(rootDir.absolutePath)
+        from('src/test/configs')
+        into 'build/configs'
+    }
+
+    task copyCustomBundle(type: Copy) {
+        from("${rootDir.absolutePath}/impersonation-filter-bundle-impersonation-filter-bundle/build/libs") {
```

Comment:
I think you want to copy the output of the project build instead of doing a files copy. This way you can take advantage of the project build dependency: 

```groovy
task copyCustomBundle(type: Copy, dependsOn: project(":impersonation-filter-bundle-impersonation-filter-bundle"))
```

At the least this will ensure that project's artifacts are built before trying to copy them. I think there's a way to drill further down and properly do this. like within the task you can specify inputs being:

```groovy
project(":impersonation-filter-bundle-impersonation-filter-bundle").artifacts
```

to grab the result of a project. This way you get to take advantage of gradles knowledge of built items so it will automatically, incrementally build your project.